app/webpacker/components/SearchWidget/MultiSearchInput.jsx

```diff
@@ -67,6 +82,7 @@ export default function MultiSearchInput({
   goToItemUrlOnClick = false,
   placeholder,
   removeNoResultsMessage,
+  model,
```

Comment:
You shouldn't have to pass in the model separately.

Reply:
Removed